File: data/StarCoder2/readable_format_HPEC/fortran/file211.f90
subroutine print_array(n, y)
implicit none

DATA_TYPE, dimension(n) :: y
integer :: n
integer :: i
do i = 1, n
write(0, DATA_PRINTF_MODIFIER) y(i)
if (mod(i - 1, 20) == 0) then
write(0, *)
end if
end do
end subroutine

File: data/gpt-4o/readable_format_HPEC/cpp/file211.cpp
#include <iostream>
typedef int DATA_TYPE;
#define DATA_PRINTF_MODIFIER "%d "

void print_array(int n, DATA_TYPE y[]) {
 for (int i = 0; i < n; i++) {
  printf(DATA_PRINTF_MODIFIER, y[i]);
  if (i % 20 == 0) {
   printf("
");
  }
 }
}
